Source/WebCore/editing/DeleteSelectionCommand.cpp

```diff
@@ -303,9 +303,9 @@ bool DeleteSelectionCommand::initializePositionData()
     initializeStartEnd(start, end);
     
     if (!isEditablePosition(start, ContentIsEditable))
-        start = firstEditablePositionAfterPositionInRoot(start, highestEditableRoot(start));
+        start = firstEditablePositionAfterPositionInRoot(start, RefPtr { highestEditableRoot(start) }.get());
```

Comment:
Should we just change these functions to return `RefPtr`? What can we do with a raw pointer without getting a static analyzer warning?